compiler/bundler.rs

```diff
@@ -0,0 +1,212 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+use crate::source_map_bundler::SourceMapBundler;
+use crate::Result;
+
+use deno_core::ErrBox;
+use std::error::Error;
+use std::fmt;
+use std::path::PathBuf;
+
```

Comment:
Sooo... [`swc_bundler`](https://crates.io/crates/swc_bundler) is now available and I wanted to try and incorporate it into our workflow, but I guess we'll start with your version first and then try to switch to it. WDYT?